addons/base_vat/models/res_partner.py

```diff
@@ -63,6 +66,8 @@
     'be': 'BE0477472701',
     'bg': 'BG1234567892',
     'ch': 'CHE-123.456.788 TVA or CH TVA 123456',  # Swiss by Yannick Vaucher @ Camptocamp
+    'cl': 'CL76086428-5',
+    'co': 'CO213123432-1 or CO213.123.432-1',
```

Comment:
aren't we removing the need of the prefix CO-CL? in México we do not use suech validation since V11....

Reply:
It already works like this.  (even if not in the examples)